examples_tests/61.UI/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+include(common RESULT_VARIABLE RES)
+if(NOT RES)
+	message(FATAL_ERROR "common.cmake not found. Should be in {repo_root}/cmake directory")
+endif()
+
+set(EXTRA_SOURCES "")
+list(APPEND EXTRA_SOURCES 
+	"UI_System.cpp" 
+	"UI_System.hpp"
+)
```

Comment:
you want to turn this into an extension `nbl::ext::imgui`

Reply:
see other extensions to see how they're supposed to look

Reply:
Done